### ISSUE TITLE
IdsToolbar: Not able to click nested menu item when overflowed

### ISSUE THREAD

**Issue by whernebrink** [infor-design/enterprise-wc#930]
**Describe the bug**
Not able to click (or navigate by keyboard) on a nested menu (sub menu) item, if the menu has been automatically moved  to the more button (haven't tested if it works when then nested menu was purposely added to the more menu e.g. you only have a more button with items).

**To Reproduce**
1. Go to https://main.wc.design.infor.com/ids-toolbar/overflowed.html
2. Make screen size smaller so that `Text Button 1` and `Menu Button 2` is hidden in the more menu.
3. Try to click the `Menu Button 2 > More Items > Item x` now
4. Then menu will hide when trying to click the nested menu, and you cannot use keyboard navigation for the overflowed menu. Also, the `has-icon` class has now been added, which applies wrong styling I would assume.

**Expected behavior**
The menu should not be closed and one should be able to click the menu items and be able to use keyboard navigation.

**Version**
- ids-enterprise-wc: 1.0.0-beta.1? (couldn't see a version on the html, but whatever is on main.wc.design today)

**Screenshots**
![wc-toolbar](https://user-images.githubusercontent.com/73601152/190523368-07c8c38b-a770-410f-9279-d7b179cb78f6.gif)

The class `has-icon` is also added, even though there is no icon, which gives it wrong styling:
![image](https://user-images.githubusercontent.com/73601152/190523900-09dc29f1-3da8-48a5-bee3-6474f0ddcfca.png)

**Platform**
 - Infor M3/Experience Designer
 - OS Version:  osMonteray
 - Browser Name: Firefox
 - Browser Version: 104.0.2

**Comment by tmcconechy** [comment]
Also noticed that the icon buttons look funny (icon on the wrong size and padding?)

**Comment by whernebrink** [comment]
Yeah, notixed that too. When trying in my angular stackblitz, you got different result depending on the order you put the span and the icon:

![image](https://user-images.githubusercontent.com/73601152/190690114-94e34eaf-ca19-4cae-bc0b-906ba05c7785.png)

**Comment by EdwardCoyle** [comment]
Just noting here that we've already raised #839 for the button issues
